modules/tensor_mechanics/include/auxkernels/ComputeValueRate.h

```diff
@@ -0,0 +1,25 @@
+/*
+AuxKernel of Passing Variable Time Derivative
+*/
+
+#pragma once
+
+#include "AuxKernel.h"
+
+class ComputeValueRate : public AuxKernel
```

Comment:
we have a TimeDerivativeAux

Reply:
sure

Reply:
TimeDerivativeAux seems only work for elemental aux, here I would like to take derivative of nodal aux, is there a workaround to do this?

Reply:
I have a second thought on this. Since nodal velocity value only acts as post-processing quantity, I could delete this auxkernels without affecting the calculation

Reply:
You ll have to delete it anyway because while it's correct, we already have the TimeDerivativeAux

Reply:
> TimeDerivativeAux seems only work for elemental aux, here I would like to take derivative of nodal aux, is there a workaround to do this?

Based on the test, it currently only for elemental aux variable